### PR TITLE
Hotfix/seperate adress field

### PR DESCRIPTION
**Pull request by martinyde**
https://jira.itkdev.dk/browse/JE-227

- Adskil adresse of info felter fra taksonomi termer og tilføj felter på høringer.
- Migrér indhold fra taksonomi ud på noderne med drush kommando.

### CODE REVIEW

File: web/modules/custom/migrate_subject_data/composer.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+      "Drupal\\migrate_subject_data\\": "src/"
+    }
+  },
+  "extra": {
+    "drush": {
+      "services": {
+        "drush.services.yml": "^9"
+      }
+    }
+  }
```
Comment (rimi-itk) [inline web/modules/custom/migrate_subject_data/composer.json]: Out of curiosity: What does this do?

Comment (martinyde) [reply web/modules/custom/migrate_subject_data/composer.json]: "The drush.services.yml file needs to be declared in your composer.json for Drush 9 commands to work properly. Here, you can also define multiple services files to be used for different versions of Drush. In Drush 9, defining the services files explicitly in the composer.json file is not mandatory, and if the file is not defined, the default drush.services.yml file will be loaded.

However, from Drush 10 onwards, the services section in the composer.json file must exist, and it should name the services file to be used. Therefore, it is recommended to declare the services file in Drush 9 too." - https://www.axelerant.com/resources/team-blog/how-to-write-custom-drush-9-commands-for-drupal-8

File: config/sync/core.entity_view_display.node.hearing.hearing_ticket_view.yml [hunk 2]
```diff
@@ -5,17 +5,20 @@ dependencies:
   config:
     - core.entity_view_mode.node.hearing_ticket_view
     - field.field.node.hearing.field_area
+    - field.field.node.hearing.field_contact
     - field.field.node.hearing.field_content_state
     - field.field.node.hearing.field_description
     - field.field.node.hearing.field_deskpro_agent_email
     - field.field.node.hearing.field_deskpro_data
     - field.field.node.hearing.field_deskpro_department_id
+    - field.field.node.hearing.field_edoc_casefile_id
```
[the remark this answers is not in the document]
Comment (martinyde) [reply config/sync/core.entity_view_display.node.hearing.hearing_ticket_view.yml]: Yeah, i'd assume so. I did a full export after a db sync, so the db expects it to exist.

**Comment by martinyde** [comment]
> @martinyde, where does “JE-227” come from? Do you mean “HRIN-333”?

Year, thats an error 333 is correct

**Pull request merged**